test/acceptance/features/unbindAppToService.feature

```diff
@@ -247,3 +247,129 @@ Feature: Unbind an application from a service
         * File "/bindings/spec-remove-bindings-as-files-app-sb/port" is unavailable in application pod
         * File "/bindings/spec-remove-bindings-as-files-app-sb/type" is unavailable in application pod
         * Service Binding secret is not present
+
+    Scenario: Bind without SERVICE_BINDING_ROOT set, unbind and re-bind
```

Comment:
what is the purpose of this tests, i.e. expectations? How it is related to the submitted fix?

Reply:
Those tests should verify various situations where SERVICE_BINDING_ROOT is or not set on the application. But after deeper consideration, I can see that those cases are covered by the other tests after adding the additional step for checking number of env occurencies... so removing this from PR

Thanks